### DISCUSSION TITLE
SKR2 & TFT35 V3 config

### DISCUSSION THREAD

**Discussion by Dreamwebb** [teeminus/NoTouchScreenFirmware#53] [Q&A]
Does anyone have a working config for a SKR 2 + TFT 35 V3 Combo?

I've tried looking at the pinouts of the SKR 2 and the TFT35 but im not really sure what pins i need for the parameters.

Would appreciate if anyone could help out :)

cs_pin:
sclk_pin: 
sid_pin: 
encoder_pins: 
click_pin: 

[TFT35 V3.0-PIN.pdf](https://github.com/teeminus/NoTouchScreenFirmware/files/7118675/TFT35.V3.0-PIN.pdf)
[BIGTREETECH SKR 2-Pin.pdf](https://github.com/teeminus/NoTouchScreenFirmware/files/7118676/BIGTREETECH.SKR.2-Pin.pdf)
...

**Comment by teeminus** [comment]
Sorry for the late reply :(

Have you searched the internet (e.g. in the original BTT firmware repo) for a display config for your mainboard?

**Reply by Dreamwebb** [reply]
the closest thing i found was in the SKR 2 example configuration for klipper

but im not sure which pins match up for the st7920 cofiguration

[board_pins]

aliases:

    # EXP1 header
    EXP1_1=PC5, EXP1_3=PB1, EXP1_5=PE10, EXP1_7=PE12, EXP1_9=<GND>,
    EXP1_2=PB0, EXP1_4=PE9, EXP1_6=PE11, EXP1_8=PE13, EXP1_10=<5V>,
    # EXP2 header
    EXP2_1=PA6, EXP2_3=PE7, EXP2_5=PB2, EXP2_7=PC4,   EXP2_9=<GND>,
    EXP2_2=PA5, EXP2_4=PA4, EXP2_6=PA7, EXP2_8=<RST>, EXP2_10=<NC>

**Reply by teeminus** [reply]
If you've got the EXP placeholders, the sample st7920 display config from the Klipper repo should work.

**Reply by Dreamwebb** [reply]
which example in the https://github.com/KevinOConnor/klipper/blob/master/config/sample-lcd.cfg

theres 2 samples for the st7920

**Reply by teeminus** [reply]
You could try both. If none is working you could have a look in the wiki of this repo for more config examples.

**Reply by Dreamwebb** [reply]
ok i got it working with this configuration
lcd_type: st7920
cs_pin: EXP1_7
sclk_pin: EXP1_6
sid_pin: EXP1_8
encoder_pins: ^EXP1_5, ^EXP1_3
click_pin: ^!EXP1_2

im guessing theres no way to auto start the emulation? i have to hold my selector knob in for a few seconds get past the ST7920 emulator ready screen

**Comment by drak42** [comment]
Following this as I'm also looking for this information
